Minor changes for porting WW3 for GEFSv12 to WCOSS2

XianwuXue-NOAA opened the pull request:
# Pull Request Summary
This PR is to delete option "-fast-transcendentals" in WW3

## Description
Minor change to delete option "-fast-transcendentals" to keep the precise  implementations as the current production (DELL35)

### Issue(s) addressed
* WW3 issue: #445 (Port to WCOSS2 for GEFSv12)
* Reviewers:  @JessicaMeixner-NOAA 

### Commit Message
* Use existing commit message if possible//no merge commit

### Testing
* How were these changes tested? Tested in GEFSv12 on WCOSS2

XianwuXue-NOAA committed:
Port to WCOSS2/Acorn

 On branch feature/gefs_v12_port2acorn
	modified:   model/bin/cmplr.env
	modified:   model/bin/w3_setup
	modified:   model/esmf/Makefile

Refs: #445

JessicaMeixner-NOAA committed:
update esmf makefile

JessicaMeixner-NOAA committed:
update cmplr.env to make sure we are using the right settings for wcoss2

JessicaMeixner-NOAA committed:
make sure wcoss2 is not using any xhost

JessicaMeixner-NOAA committed:
try flags suggested for optimization

XianwuXue-NOAA committed:
Merge remote-tracking branch 'jessica/gefsv12/wcoss2port_optimizationflag' into HEAD

XianwuXue-NOAA committed:
Fix typo in the added flags suggested for opt

 Changes to be committed:
	modified:   model/bin/cmplr.env
Refs: #445

XianwuXue-NOAA committed:
Merge branch 'production/GEFS.v12' into feature/gefs_v12_port2acorn

XianwuXue-NOAA committed:
Remove fast-transcendentals to keep the precise

Remove -fast-transcendentals option to keep its precise
	implementations in  Production

 On branch feature/gefs_v12_port2acorn
 Changes to be committed:
	modified:   model/bin/cmplr.env

Refs: #445

JessicaMeixner-NOAA commented on model/bin/cmplr.env:
```diff
@@ -147,7 +147,8 @@ if [ "$cmplr" == "intel" ]          || [ "$cmplr" == "intel_debug" ]    || [ "$c
     optl="$optl -xHOST"
   elif [ ! -z "$(echo $cmplr | grep wcoss2)" ] ; then
-    optc="$optc -fp-model precise -ftz -fast-transcendentals -no-prec-div -no-prec-sqrt -align array64byte"
+    # optc="$optc -fp-model precise -ftz -fast-transcendentals -no-prec-div -no-prec-sqrt -align array64byte"
```
Is there any reason to keep this line commented out?

XianwuXue-NOAA replied:
I want to keep the original one for the future reference. However, I am OK to delete this statement if you think it is not needed.

XianwuXue-NOAA commented:
@JessicaMeixner-NOAA After the comparison of the results (details in the email conversion and Dingchen's figures), we decide to delete  "-fast-transcendentals" in all WW3, GSDCHEM and fv3 to keep the precise implementations as the current production.

JessicaMeixner-NOAA commented:
@XianwuXue-NOAA sounds good.  I've been following the email.  Just have one question on if we should go ahead and clean-up that line with it commented out.  Once that's been decided we can merge this.

XianwuXue-NOAA commented:
No problem. I will delete it now and put to GitHub.

XianwuXue-NOAA committed:
Clean up unused statement

 On branch feature/gefs_v12_port2acorn

 Changes to be committed:
	modified:   model/bin/cmplr.env

Refs: #445
